lms/views/api/exceptions.py

```diff
@@ -110,7 +113,10 @@ def proxy_api_access_token_error(self):
 
     @exception_view_config(context=CanvasAPIPermissionError)
     @exception_view_config(context=CanvasFileNotFoundInCourse)
-    def canvas_api_permission_error(self):
```

Comment:
Not really `permissions` related anymore with `CanvasFileNotFoundInCourse` already there